module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -46,6 +46,10 @@
  */
 class EDS extends DefaultRecord
 {
+    use Feature\IlsAwareTrait {
+        Feature\IlsAwareTrait::getRealTimeHoldings as getRealTimeHoldings;
```

Comment:
I'm surprised PHP lets you alias this method name to itself -- if the goal is to "get it out of the way" I wonder if it's better to give it a different name:

```suggestion
        Feature\IlsAwareTrait::getRealTimeHoldings as getRealTimeHoldingsTraitVersion;
```

...though ideally it might be better still to make the trait itself more flexible so we don't need to override the method here but could instead set some properties (or something like that).

Reply:
This was down to me not correctly understanding the traits feature of PHP - I appreciate that making the trait more flexible might be the way to go in the future, but for now I might simple change this to `use Feature\IlsAwareTrait`